_data/v9-toc.yml

```diff
@@ -21,6 +21,8 @@ toc:
         url: /ibm-config
       - page: Configuration reference
         url: /config-definition
+      - page: Restic Install
+        url: /restic#setup
```

Comment:
Could this both be Install/#install or Setup/#setup?

Reply:
Updated to match